### PR TITLE
Fixes #338 - release prep

### LINKED ISSUE #338: Release prep
// Title: Provide a short but descriptive summary of the issue.

// **Do not include requests for development or report issues with code or products here.** 
This repository contains documentation only.

## To do

- [x] update the releases and versioning table -- add marathon-bigip-ctlr v1.2
- [x] update controller image versions in config examples
- [x] update lb algorithms in config examples: Some config examples may reference a type of lb algorithm that's not supported by the Controller in question. Audit the examples and update the lb mode as needed.
- [x] fix for SR 1-3760644427
- [x] add deprecated version notice to v1 docs

### Environment

- [ x] Cloud Foundry
- [ x] Kubernetes
- [ x] OpenShift
- [ x] Mesos/Marathon)

### Use Case 
(i.e., what are you trying to do that you can't find documentation for?)

### LINKED ISSUE #342: Remove "v2.0" from link text
Please remove "v2.0" from link text just about the documentation search box.  An example can be found on this page (as well as on attached screenshot):

http://clouddocs.f5.com/containers/v2/kubernetes/

Currently, the link text shows "F5 Container Integrations v2.0" which implies that it is the version of the F5 container integrations - which it is not - and will confuse customers. 

If the "v2.0" is supposed to be in reference to the version of the documentation, it needs to be define in a more subtle way and in a different location.

screen shot 2018-01-26 at 9 47 06 am

Thanks

### PR DESCRIPTION
**Pull request by jputrino**
@sjsharma2001 @amudukutore 

#### What issues does this address?
Fixes #338
Fixes #342 

@dgarrisonf5 

#### What's this change do?
- Includes documentation updates for
  - k8s-bigip-ctlr v1.4.0
  - cf-bigip-ctlr v1.1.0
  - marathon-bigip-ctlr v1.2.0

#### Where should the reviewer start?
View the [staged documentation](https://s3-us-west-2.amazonaws.com/staging.clouddocs.f5.com/containers/pr340/index.html)

### CODE REVIEW

File: docs/kubernetes/flannel-bigip-info.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+Overview of Cluster Networking with flannel in Kubernetes
+---------------------------------------------------------
+
+.. sidebar:: :fonticon:`fa fa-info-circle` Related
+
+   Read about the Kubernetes `Cluster Network`_ and `Using flannel with Kubernetes`_.
+
+`Flannel`_ is a layer 3 network fabric (or, in their words, "a virtual network that attaches IP addresses to containers"). Flannel assigns a subnet to each Node and allocates an IP address within the Node's subnet to each of the Node's Pods.
+
+For Kubernetes v1.6 and later, flannel provides a ``kubeadm`` network add-on. When you apply the `flannel manifest`_ in Kubernetes, the flannel Pod deploys on each Node in the Cluster. The Pod consists of two containers:
```
Comment (rsalsa) [inline docs/kubernetes/flannel-bigip-info.rst]: Can we remove the kubeadm reference? The flannel manifest file can be used without kubeadm. There is no need to reference it here. Everything else in the paragraph is fine.

File: docs/kubernetes/kctlr-use-bigip-k8s.rst [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+      admin@BIG-IP(cfg-sync Standalone)(Active)(/Common)(tmos)$ list net tunnels tunnel flannel_vxlan
+
+
+.. _k8s-flannel-setup:
+
+Deploy flannel for Kubernetes
+-----------------------------
+
+Most users deploy flannel as a `network addon`_ when setting up ``kubeadm``. See Kubernetes` :k8sdocs:`Installing a Pod network <setup/independent/create-cluster-kubeadm/#pod-network>` guide for instructions.
```
Comment (rsalsa) [inline docs/kubernetes/kctlr-use-bigip-k8s.rst]: Again, let's not bring kubeadm into the conversation as there are other methods for installing k8s and certified distributions.

File: docs/kubernetes/kctlr-use-bigip-k8s.rst [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+**If you want to install flannel manually**, take the steps below.
+
+#. (Optional) Edit the flannel `kube-flannel manifest`_.
+
+   - ``Backend``: Defaults to ``vxlan``; see `Backends <https://github.com/coreos/flannel/blob/master/Documentation/backends.md>`_ for additional information.
```
Comment (rsalsa) [inline docs/kubernetes/kctlr-use-bigip-k8s.rst]: We should not mention other backends as we do not support them (yet).

File: docs/kubernetes/kctlr-use-bigip-k8s.rst [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+   \
+
+.. seealso::
+
+   For more information about flannel and Kubernetes, see `Using flannel with Kubernetes`_.
+
+#. Verify flannel deployment.
+
+   Check the Node resource(s) for the following Annotations: ::
```
Comment (rsalsa) [inline docs/kubernetes/kctlr-use-bigip-k8s.rst]: Should we include the example kubectl command to guide the user on how to get this?

File: docs/cloudfoundry/cf-per-route-virtuals.rst [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+
+.. literalinclude:: /cloudfoundry/config_examples/manifest.yaml
+   :linenos:
+   :caption: Example App Manifest for cf-bigip-ctlr Service Broker
+
+:fonticon:`fa fa-download` :download:`Download the example manifest </cloudfoundry/config_examples/manifest.yaml>`
+
+.. _add-remove service plans:
+
+Add/Remove Service Plans
```
[the remark this answers is not in the document]
Comment (jputrino) [reply docs/cloudfoundry/cf-per-route-virtuals.rst]: Done. I did a bit of a rewrite on this doc to make it easier to follow.

[the remark this answers is not in the document]
Comment (rsalsa) [reply docs/cloudfoundry/cf-per-route-virtuals.rst]: staged docs update for latest revision?

**Comment by jputrino** [comment]
@rsalsa - staged docs are up-to-date

**Pull request merged**